lib/msal-node/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://json.schemastore.org/package.json",
   "name": "@azure/msal-node",
-  "version": "2.16.2",
+  "version": "3.0.0",
```

Comment:
Do we want to remove the deprecated node we had to add back in as a rush after the last major release?

Reply:
Yeah we can. Up to you if you want to wrap it up in this or vNext.